.github/workflows/release.yaml

```diff
@@ -30,35 +34,37 @@ jobs:
         with:
           node-version: 18
           registry-url: 'https://registry.npmjs.org'
+      - name: Determine new template version
+        run: echo "VERSION=$(./scripts/bumpedTemplateVersion.sh \"${{ inputs.version }}\")" >> $GITHUB_ENV
```

Comment:
wouldn't be better to have the script in JS instead of bash? It's more readable, let us use higher level functions, and it allow us to write tests if we want to.

Reply:
+1

Reply:
Agreed, I'm more inclined to that the longer we work on these.